scripts/run.py

```diff
@@ -84,11 +93,16 @@ def train_model(config, model, train_path):
                                                      '--vocab_save', vocab_path,
                                                      '--n_jobs', str(config.n_jobs)])[0]
     trainer_script.main(model, trainer_config)
+'''
+
 
 def sample_from_model(config, model):
-    model_path = get_model_path(config, model)
-    config_path = get_config_path(config, model)
-    vocab_path = get_vocab_path(config, model)
+    #model_path = get_model_path(config, model)
+    model_path = config.model_path
```

Comment:
Does this work, if LBANN is not used?

Reply:
This change makes the code more flexible. For example:
```
model_path = get_model_path(config, model)
```

will call this:

```
def get_model_path(config, model):
    return os.path.join(config.checkpoint_dir, model + config.experiment_suff + '_model.pt')
```
as opposed to this:

```
model_path = config.model_path
```

where the user just defines the model_path themselves.

```
parser.add_argument('--model-path', type=str, default='',
                        help='path to experiment model')
```